infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -474,7 +471,7 @@ async function initializeWethTokenForHyperchain() {
 async function startServer() {
     const YES_DEFAULT = 'Yes (default components)';
     const YES_CUSTOM = 'Yes (custom components)';
-    const NO = 'Not right now';
+    const NO = 'Not right now (you want to use prover with docker-compose)';
```

Comment:
prover = server?

Reply:
Yes, but no =) This prompt _before_ we setup provers or docker compose, and we have all necessary parts (including server) in docker-compose file. So to prevent user's confusion (how did he know what all he needed will be generated on later step), we are asking him "Are you planing run prover? Select no". Maybe wording can be better =)

Reply:
Then let's change to "Not right now (you can now configure prover, generate docker files, or just run the server later)"

Reply:
Done!